hws/mlx_hw_steering_parser.py

```diff
@@ -244,30 +248,33 @@ def parse_args():
 
     _config_args["extra_hw_res_arg"] = False
     _config_args["extra_hw_res_pat"] = False
+    _config_args["extra_hw_res_counter"] = False
 
     if (args.dump_hw_resources):
         _config_args["dump_hw_resources"] = True
         if (args.device == ""):
             _config_args["device"] = None
         else:
             _config_args["device"] = args.device
-
-        for hw_res in args.extra_hw_res.split(","):
-            if hw_res == "all":
-                _config_args["extra_hw_res_pat"] = True
-                _config_args["extra_hw_res_arg"] = True
-                break
-            elif hw_res == "pat":
-                _config_args["extra_hw_res_pat"] = True
-            elif hw_res == "arg":
-                _config_args["extra_hw_res_arg"] = True
-
-        if _config_args.get("extra_hw_res_arg") and not(_config_args.get("extra_hw_res_pat")):
-            _config_args["extra_hw_res_arg"] = False
-
     else:
         _config_args["dump_hw_resources"] = False
 
+    for hw_res in args.extra_hw_res.split(","):
```

Comment:
You took it out from under the 'dump_hw_resource'. Any special reason? Because now you can specify it combined with '--skip_dump'.

Reply:
yes, as i see this case was missing (to use with --skip_dump) if we have already the dump and just want to load